### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript-contrib/governance#35

### CODE REVIEW

**Comment by JordanMartinez** [comment]
I'd like someone familiar with React to double-check the kind signatures I added to the classes. If these should have a concrete kind of Type, let me know.

File: src/React.purs [hunk 1]
```diff
@@ -170,15 +170,17 @@ type ReactClassConstructor props state r =
   ReactThis props state ->
   Effect (Record r)
 
-class ReactComponentSpec props state snapshot (given :: # Type) (spec :: # Type)
+class ReactComponentSpec :: forall k1 k2 k3. k1 -> k2 -> k3 -> Row Type -> Row Type -> Constraint
```
Comment (thomashoneyman) [inline src/React.purs:173]: These can be `Type`.

File: src/React.purs [hunk 2]
```diff
@@ -170,15 +170,17 @@ type ReactClassConstructor props state r =
...
+class ReactComponentSpec props state snapshot (given :: Row Type) (spec :: Row Type)
 
 instance reactComponentSpec ::
   ( Row.Union given (ReactSpecAll props state ReactUnusedSnapshot) spec
   , Row.Nub spec (ReactSpecAll props state snapshot)
   ) =>
   ReactComponentSpec props state snapshot given spec
 
-class ReactPureComponentSpec props state snapshot (given :: # Type) (spec :: # Type)
+class ReactPureComponentSpec :: forall k1 k2 k3. k1 -> k2 -> k3 -> Row Type -> Row Type -> Constraint
```
Comment (thomashoneyman) [inline src/React.purs:182]: These can be `Type` as well.

Comment (JordanMartinez) [reply src/React.purs]: Fixed in latest commit

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**